e2e/davinci-suites/playwright.config.ts

```diff
@@ -17,14 +17,23 @@ const config: PlaywrightTestConfig = {
     trace: process.env.CI ? 'on-first-retry' : 'retain-on-failure',
   },
   webServer: [
+    process.env.CI == 'false'
```

Comment:
added this for the watch-deps so that changes are reloaded when running playwrights ui mode